src/notebook/notebookFeature.ts

```diff
@@ -5,14 +5,35 @@ import { WorkspaceFeature } from '../interactive/workspace';
 import { getJuliaExePaths, JuliaExecutable } from '../juliaexepath';
 import { JuliaKernel } from './notebookKernel';
 
+const JupyterNotebookViewType = 'jupyter-notebook';
+type JupyterNotebookMetadata = Partial<{
```

Comment:
Currently there are no type definitions for this, 
structure of the metadata. Same approach used by .NEt interactive extension as well.
I'm working on this (trying to get a typed approach to accessing this metadata)